packages/vats/src/lib-board.js

```diff
@@ -28,28 +61,112 @@ const calcCrc = (data, crcDigits) => {
 };
 
 /**
- * Create a board to post things on.
+ * Make board state init function, closing over baggage.
  *
- * @param {bigint | number} [initSequence]
- * @param {object} [options]
- * @param {string} [options.prefix]
- * @param {number} [options.crcDigits]
+ * @see {prepareExoClassKit}
+ * @see {@link packages/SwingSet/docs/virtual-objects.md|SwingSet Virtual Objects}
```

Comment:
what's the pipe meant to do?

Reply:
this would make the line clickable in IDE hover.
```suggestion
 * @see {@link [virtual-objects](../..//SwingSet/docs/virtual-objects.md)}
```
I hoped VS Code would also be smart enough to update that path when the filename changes, but nay

Reply:
The pipe comes from an example in [@see docs](https://jsdoc.app/tags-see.html):

```
 * @see {@link http://github.com|GitHub}
```

Reply:
Ah. VS Code doesn't observe that:
<img width="243" alt="Screenshot 2023-02-02 at 9 19 15 AM" src="https://user-images.githubusercontent.com/21505/216396096-33469e44-5d20-423d-a9d2-f5133daeba66.png">

TSDoc hasn't settled on a syntax: https://github.com/microsoft/tsdoc/issues/9

Fine with whatever. We'll probably clean up all the links when we adopt a doc generator.

Reply:
re doc generator, for back-link indexing:

 - #4291